docs/metrics/accelerate.md

```diff
@@ -92,6 +94,8 @@ Pulse uses the average value over a period of time to display aggregate change f
 
         This means that you must report both deployments and incidents associated with the correct systems for Pulse to calculate the metrics correctly. Otherwise, even if you're reporting incident events, the charts will be empty.
```

Comment:
Same as above

```suggestion
        This means that you must report deployments and incidents associated with the correct systems for Pulse to calculate the metrics correctly. Otherwise, even if you're reporting incident events, the charts will be empty.
```